data/pizza.json

```diff
@@ -1095,7 +1095,7 @@
 			"hidden_ingredients": [
 				"bröd"
 			],
-			"price": 85
+			"price": 93
 		},
 		"grekisk salladsburgare": {
```

Comment:
[Vitlökssås instead of valfri sås](https://sannegarden.com/meny/johanneberg/#:~:text=89Kr-,L%C3%B6k%2C%20Oliver%2C%20Vitl%C3%B6kss%C3%A5s%2C%20F%C3%A5rost%2C%20Gurka%2C%20Isbergssallad%2C%20Majs%2C%20Tomater,-Sallader)